### ISSUE TITLE
onAnimationUpdate run even if the view is invisble when set autoPlay=true

### ISSUE THREAD

**Issue by GeeJoe** [airbnb/lottie-android#1168]
Recently，I found my app's rendering performance was very poor, After debugging, I found that a Lottie animation was running endless. Finally I found out that an invisible LottieAnimationView was running which was inconsistent with my expectations.

Here is my code 

<com.airbnb.lottie.LottieAnimationView
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="40dp"
    android:layout_height="40dp"
    android:visibility="gone"
    app:lottie_autoPlay="true"
    app:lottie_loop="true"
    app:lottie_rawRes="@raw/list_loading"/>

I set the visibility="gone" and lottie_autoPlay="true"，then I debug in android studio by putting a breakpoint at BaseLottieAnimator#notifyUpdate(), I found the method loop endless which means the animation is running even I set the view's visibility to GONE

I think this may be a bug and hope that some one can fix it.

**Comment by gpeal** [comment]
@GeeJoe This should be fixed with 3.0. Can you upgrade and try again? Feel free to reopen and attach a sample project if that isn't the case.

**Comment by GeeJoe** [comment]
yes, It was already the 3.0. version.
I saw that it fixed the problem by override the onVisibilityChanged and pause the animation when the view is invisible.It worked perfectly when I set the view's visibility dynamically later. But when I set android:visibility="gone" in the xml and lottie_autoPlay="true", the animation was still running as soon as the view is inflated.

I think that it may won't invoke the onVisibilityChanged when i set the visibility in xml
